docs/edgeql/operators/json.rst

```diff
@@ -59,9 +93,50 @@ The contents of JSON *objects* and *arrays* can also be accessed via ``[]``:
     db> SELECT to_json('[1, "a", null]')[-1];
     {None}
 
-    db> SELECT j := <json>(schema::Object {
+The index access operator ``[]`` will raise an exception if the
+specified index is not valid for the base JSON value. To access
+potentially out of bound indexes use the :eql:func:`json_get`
+function.
+
+
+Slicing JSON Arrays
+===================
+
+JSON arrays can be sliced in the same way as regular arrays, producing
+a new JSON array:
+
+.. code-block:: edgeql-repl
+
+    db> SELECT to_json('[1, 2, 3]')[0:2];
+    {[1, 2]}
```

Comment:
` {[1, 2]}` -> `{'[1, 2]'}`

Please also update all other examples

Reply:
I'd rather do it when the repl *actually* produces that output. Because then I can use repl to actually verify that what I get is really what I claim.